src/ledger/v1/blockchain_ledger_v1.erl

```
@@ -3303,22 +3425,33 @@ get_aux_rewards_(Itr, {ok, Key, BinRes}, Acc) ->
         end,
     get_aux_rewards_(Itr, rocksdb:iterator_move(Itr, next), NewAcc).
 
+-spec validators_cf(ledger()) -> rocksdb:cf_handle().
+validators_cf(Ledger) ->
+    SL = subledger(Ledger),
+    SL#sub_ledger_v1.validators.
+
 -spec cache_put(ledger(), rocksdb:cf_handle(), binary(), binary()) -> ok.
 cache_put(Ledger, CF, Key, Value) ->
-    {Cache, _GwCache} = context_cache(Ledger),
-    true = ets:insert(Cache, {{CF, Key}, Value}),
-    ok.
+    case context_cache(Ledger) of
+        {undefined, _GwCache} -> ok; %% needed for test cases
```

I'm concerned this will cause silent failures, why do the test cases need it?

---

@mrallen1 added this, I think, maybe he can clarify?

---

we can remove it I think - at the time I was worried about a test failure in the unstake height PR which was triggered by this case being unhandled